src/main.rs

```diff
@@ -211,6 +211,33 @@ fn run_machete() -> anyhow::Result<bool> {
         }
     }
 
+    if has_unused_dependencies {
+        if !args.with_metadata {
+            println!("\n\
```

Comment:
nit: we could use a raw string here:

```rust
println!(r#"my

long

string");
```

Reply:
I tried, but the formatting was very unpleasant with a raw string, especially after cargo fmt got its hands on it.